doc/classes/Object.xml

```diff
@@ -153,18 +181,73 @@
 				A [code]signal[/code] can only be connected once to a [code]method[/code]. It will throw an error if already connected, unless the signal was connected with [constant CONNECT_REFERENCE_COUNTED]. To avoid this, first, use [method is_connected] to check for existing connections.
 				If the [code]target[/code] is destroyed in the game's lifecycle, the connection will be lost.
 				Examples:
-				[codeblock]
-				connect("pressed", self, "_on_Button_pressed") # BaseButton signal
-				connect("text_entered", self, "_on_LineEdit_text_entered") # LineEdit signal
-				connect("hit", self, "_on_Player_hit", [ weapon_type, damage ]) # User-defined signal
-				[/codeblock]
+				[codeblocks]
+				[gdscript]
+				var button = Button.new()
+				button.pressed.connect(_on_Button_pressed) # is the same as:
```

Comment:
There's a huge `FIXME` yelling at the beginning of this description... :)

Reply:
Note that the `connect` that you're using in this line is not the same one as `Object.connect` which you're documenting. It's actually `[method Signal.connect]` and this should be made clear to the user.

Reply:
I have a little trouble phrasing that. I wrote: `We use [method Signal.connect] in the examples:` instead of `Examples:`

Reply:
The FIXME did imply actually fixing the code, this code doesn't compile. The descriptions are also wrong (no `method` or `target`, it's now taking a `callable`). It's better to leave the FIXME if it's not properly tested.